azurerm/data_source_cosmos_db_account.go

```diff
@@ -93,6 +93,14 @@ func dataSourceArmCosmosDBAccount() *schema.Resource {
 				},
 			},
 
+			"capabilities": {
+				Type:     schema.TypeList,
```

Comment:
doesn't this also want to be a Set?

Reply:
The other Sets here were changed to Lists for the datasource, should we be using sets for them too?

I can see the value in having a list so they can be easily addressed, and not needed the deduplication of a set because they come from the API.